### PR TITLE
Breakpoint editor in sample apps

### PR DESCRIPTION
**Pull request by Rookeret73**
RK-613

### CODE REVIEW

File: docs/sample-applications.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ __Level: Beginner__
     - **Reproduce:** Add a todo with special characters. All of these characters should disappear.
     - **Debug:**
         1. In the Rookout app, open the file `app.py`
-        2. In line 66 we see that the todo title is being filtered by `cleanStr` - Let's add a `Snapshot` to line 67
+        2. In line 66 we see that the todo title is being filtered by `cleanStr` - Let's add a Snapshot Breakpointto line 67
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: space

File: docs/sample-applications.md [hunk 2]
```diff
@@ -162,24 +162,12 @@ __Level: Intermediate__
...
-        {
-            "name": "send_rookout",
-            "path": "store.rookout.locals.dup"
-        }
-        ```
-        
-        7. Add and duplicate a todo in order to see the output, and now we can see what is being given to the object and match if we have an error in the function (parameters missing or in bad order).
+        2. Using the **Breakpoint** pane on the right, add a __Snapshot__ breakpoint at line 113
+        3. Right click the Breakpoint and choose Edit from the context menu
+        4. In the Breakpoint Editor, now displayed on the right hand side, add a variable named "dup"
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: Enclose "dup" in backticks `dup` to have the code formatting

File: docs/sample-applications.md [hunk 3]
```diff
@@ -162,24 +162,12 @@ __Level: Intermediate__
...
+        5. Uncheck the "Stack Trace" and "collect all local frame variables" options, and click "Set" to save the breakpoint.  
+        6. Add and duplicate a todo in order to see the output. Note that now only the "dup" (short for duplicate) variable is fetched.
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: Enclose "dup" in backticks dup to have the code formatting

File: docs/sample-applications.md [hunk 4]
```diff
@@ -334,24 +322,12 @@ __Level: Intermediate__
...
-        {
-            "name": "send_rookout",
-            "path": "store.rookout.locals.todo"
-        }
-        ```
-        
-        7. Add and duplicate a todo in order to see the output, and now we can see what is being given to the object and match if we have an error in the function (parameters missing or in bad order).
+        2. Using the **Breakpoint** pane on the right, add a __Snapshot__ breakpoint at line 104
+        3. Right click the Breakpoint and choose Edit from the context menu
+        4. In the Breakpoint Editor, now displayed on the right hand side, add a variable named "todo"
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: Enclose "todo" in backticks`todo` to have the code formatting

File: docs/sample-applications.md [hunk 5]
```diff
@@ -334,24 +322,12 @@ __Level: Intermediate__
...
+        5. Uncheck the "Stack Trace" and "collect all local frame variables" options, and click "Set" to save the breakpoint
+        6. Add and duplicate a todo in order to see the output. Note that only the "todo" variable was fetched.
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: Enclose "todo" in backticks `todo` to have the code formatting

File: docs/sample-applications.md [hunk 6]
```diff
@@ -499,23 +475,12 @@ __Level: Intermediate__
...
-        {
-            "name": "send_rookout",
-            "path": "store.rookout.locals.dup"
-        }
-        ```
-        8. Click the save button on the upper pane.
-        9. Add and duplicate a todo in order to see the output, now we can see what is being given to the object and match if we have an error in the function (parameters missing or in bad order).  
+        2. Add a __Snapshot__ Breakpoint in the duplicateTodo function on the line with `todos.add(newTodoRecord);`
+        3. Right click the Breakpoint and choose Edit from the context menu
+        4. In the Breakpoint Editor, now displayed on the right hand side, add a variable named "newTodoRecord"
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: Enclose "newTodoRecord" in backticks newTodoRecord to have the code formatting

File: docs/sample-applications.md [hunk 7]
```diff
@@ -499,23 +475,12 @@ __Level: Intermediate__
...
+        5. Uncheck the "Stack Trace" and "collect all local frame variables" options, and click "Set" to save the breakpoint
+        9. Add and duplicate a todo in order to see the output. Note that only the newTodoRecord variable was fetched.
```
Comment (MickaelAlliel) [inline docs/sample-applications.md]: `newTodoRecord` backticks

File: docs/sample-applications.md (hunk 1 above)
Comment (Rookeret73) [reply docs/sample-applications.md]: fixed

File: docs/sample-applications.md (hunk 2 above)
Comment (Rookeret73) [reply docs/sample-applications.md]: fixed

File: docs/sample-applications.md (hunk 4 above)
Comment (Rookeret73) [reply docs/sample-applications.md]: fixed

File: docs/sample-applications.md (hunk 6 above)
Comment (Rookeret73) [reply docs/sample-applications.md]: fixed

**Comment by Rookeret73** [comment]
@MickaelAlliel Fixed and pushed

**Pull request merged**